db/primary_migrate/20221011151135_add_initiating_service_provider_id_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddInitiatingServiceProviderIdToProfile < ActiveRecord::Migration[7.0]
+  def change
+    add_column :profiles, :initiating_service_provider_id, :bigint, null: true
```

Comment:
I think we could (should?) still reference by the `issuer` string, which is what we do in a handful of other places.

For example:

https://github.com/18F/identity-idp/blob/6af52cdd6677a7d872fa63cc8d67fe58d4877924/app/models/in_person_enrollment.rb#L6-L10

Reply:
OK, after chatting with Hooper, a rationale for using the issuer is that in the event the SP is deleted, the issuer would indicate what the SP was, much better than an integer would. I'll make the change.

On a side note, I am confused as to why the example you show has `primary_key: 'issuer'` set, given that the primary key of the service_providers table is `id`, not `issuer`.

Reply:
done in 500c1983c